website/package.json

```diff
@@ -41,5 +41,10 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "@docusaurus/module-type-aliases": "^2.4.1",
+    "@tsconfig/docusaurus": "^2.0.0",
+    "typescript": "^5.1.6"
```

Comment:
I wonder if we can just move them all into the `dependency` section. I think it wouldn't make any differences.

Reply:
it's from `npm install --save-dev typescript @docusaurus/module-type-aliases @tsconfig/docusaurus`
Ref: [TypeScript Support](https://docusaurus.io/docs/typescript-support)

Reply:
Let's keep only `dependencies` or `devDependencies`. Having both might confuse folks.